PWGHF/Core/HfMlResponseDplusToPiKPi.h

```diff
@@ -0,0 +1,193 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HfMlResponseDplusToPiKPi.h
+/// \brief Class to compute the ML response for D± → π± K∓ π± analysis selections
+/// \author Alexandre Bigot <alexandre.bigot@cern.ch>, IPHC Strasbourg
+
+#ifndef PWGHF_CORE_HFMLRESPONSEDPLUSTOPIKPI_H_
+#define PWGHF_CORE_HFMLRESPONSEDPLUSTOPIKPI_H_
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "PWGHF/Core/HfMlResponse.h"
+
+// Fill the map of available input features
+// the key is the feature's name (std::string)
+// the value is the corresponding value in EnumInputFeatures
+#define FILL_MAP_DPLUS(FEATURE)                                          \
+  {                                                                      \
+#FEATURE, static_cast < uint8_t>(InputFeaturesDplusToPiKPi::FEATURE) \
```

Comment:
```suggestion
#FEATURE, static_cast <uint8_t>(InputFeaturesDplusToPiKPi::FEATURE) \
```
Is the extra space intentional?

Reply:
No :) I also noticed it, it appears after `clang-format`